iron-list.html

```diff
@@ -876,8 +876,8 @@
      * @return {boolean} True if the pool was increased.
      */
     _increasePoolIfNeeded: function() {
-      // Base case 1: the list has no size.
-      if (this._viewportSize === 0) {

```

Comment:
No for this branch, but for there're a few for `_increasePoolIfNeeded` https://github.com/PolymerElements/iron-list/blob/master/test/physical-count.html#L49